History.md

```diff
@@ -2,6 +2,9 @@
 
 ## 5.4.0 (unreleased)
 
+### Breaking Changes
+* Require Ruby >= 2.7 (#1578) [Andy Waite]
```

Comment:
Arguably this should be a major release (6.0), depending on how strict we're being with semver.

Reply:
There is already a "Changed" section with a potentially breaking change, so this should probably be combined with that section.

Reply:
I can move it, but I noticed the previous Ruby bump was listed under that section name:

```
### Breaking Changes
* Require Ruby >= 2.5 (#1131, #1310) [Mike Angell, Dylan Thacker-Smith]
```

Reply:
👍 Updated in 3bbb7aa7baba7602caa95292d809c3fbfe9d57c5